AxisState axis_requested_state
---
ActiveErrors active_errors
AxisState axis_state
ProcedureResult procedure_result
